ci/parse_trivy_results.py

```diff
@@ -131,54 +126,121 @@ def create_issue(target_name, num_vulnerabilities, severity_list, table):
         "published on Container Registry."
 
     body_args = dict(build_id=FLAGS.build_id, commit_id=FLAGS.commit_id,
-                     release_tag=FLAGS.release_tag, target_name=target_name, table=table)
+                     release_tag=FLAGS.release_tag, target_name=json_output["Target"],
+                     table=table, json_output=json.dumps(json_output, sort_keys=True, indent=4))
     body += textwrap.dedent("""
         \n
         **Cloud Build ID:** {build_id}
         **Commit ID:** {commit_id}
-        **Tag:** {release_tag}
+        **Release Tag:** {release_tag}
         **Target:** {target_name}
         ```
         {table}
-        ```""").format(**body_args)
-    print(body)
+        ```
+
+        <details>
+          <summary>JSON output from Trivy</summary>
+
+        ```json
+        {json_output}
+        ```
+        </details>""").format(**body_args)
+
+    return (title, body)
+
+
+def create_or_update_issue(repo, existing_issue, title, body):
+    """Creates or updates a Github issue for the current release."""
 
     try:
-        new_issue = repo.create_issue(title=title, body=body)
+        if existing_issue:
+            existing_issue.edit(title=title, body=body)
+            return existing_issue.number
+        else:
+            new_issue = repo.create_issue(title=title, body=body)
+            return new_issue.number
     except (BadAttributeException, GithubException) as e:
         sys.exit(
-            ("Failed to create issue due to an exception from GitHub.\nThe "
-             "error returned by GitHub API was {}").format(e))
+            ("Failed to create or update issue due to an exception from GitHub."
+             "\nThe error returned by GitHub API was {}").format(e))
+
+
+def get_duplicate_issue(repo):
+    """Finds a GitHub issue for the current release, else return None"""
+    issues = repo.get_issues(creator="ts-bridge-bot",
+                             state="open", sort="updated")
+
+    # Filter out the issue with the same release number as current build
+    title = ("Vulnerability found in release {}").format(FLAGS.release_tag)
+    prev_issue = [issue for issue in issues if issue.title.startswith(title)]
+
+    if prev_issue:
+        if len(prev_issue) > 1:
+            print("WARNING: More than one vulnerability issue exists for this "
+                  "release. We will work with the most recently updated issue.")
+        return prev_issue[0]
+    else:
+        return None
 
-    return new_issue.number
+
+def process_vulnerabilities(repo, existing_issue, trivy_json, trivy_table):
+    """Update issue and build log based on non-empty vulnerabilities."""
```

Comment:
Could you update the docstring to follow the format here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Include what the args are, and what the expected failures / returns are.